src/processor/operator/ddl/create_sequence.cpp

```diff
@@ -11,11 +11,27 @@ namespace processor {
 
 void CreateSequence::executeDDLInternal(ExecutionContext* context) {
     auto catalog = context->clientContext->getCatalog();
+    switch (info.onConflict) {
+    case common::ConflictAction::ON_CONFLICT_DO_NOTHING: {
+        if (catalog->containsSequence(context->clientContext->getTx(), info.sequenceName)) {
+            return;
+        }
+    }
+    default:
+        break;
+    }
     catalog->createSequence(context->clientContext->getTx(), info);
 }
 
 std::string CreateSequence::getOutputMsg() {
-    return stringFormat("Sequence {} has been created.", info.sequenceName);
+    switch (info.onConflict) {
+    case common::ConflictAction::ON_CONFLICT_THROW:
+        return stringFormat("Sequence {} has been created.", info.sequenceName);
+    case common::ConflictAction::ON_CONFLICT_DO_NOTHING:
```

Comment:
The output message here is a bit confusing to me. When the conflict action is `ON_CONFLICT_DO_NOTHING`, it is also possible that "sequence has been created".